processors/config/kalSimpTuple_cfg.py

```diff
@@ -76,9 +76,9 @@
 track.parameters["rawhitCollRoot"] = ''#'SCTRawHitsOnTrack_KF'
 
 #LT uncommented
-if (not options.isData):
-    track.parameters["truthTrackCollLcio"] = 'KalmanFullTracksToTruthTrackRelations'
-    track.parameters["truthTrackCollRoot"] = 'Truth_KFTracks'
+#if (not options.isData):
+#    track.parameters["truthTrackCollLcio"] = 'KalmanFullTracksToTruthTrackRelations'
+#    track.parameters["truthTrackCollRoot"] = 'Truth_KFTracks'
```

Comment:
I think you might need these to get the truth info to propagate all the way? Does it not work if you set these parameters?

Reply:
It doesn't work  if they are set: 

lcio::DataNotAvailableException: LCEventImpl::getCollection: collection not in event:KalmanFullTracksToTruthTrackRelations
Failed retrieving KalmanFullTracksToTruthTrackRelations
lcio::DataNotAvailableException: LCEventImpl::getCollection: collection not in event:GBLTracksToTruthTrackRelations
Failed retrieving GBLTracksToTruthTrackRelations

when running: 
hpstr ../src/hpstr/processors/config/kalSimpTuple_cfg.py -i /nfs/slac/g/hps2/bravo/mc/run16/simp/dev_100_60_33p3/simp_recon_simpTest.slcio -o /nfs/slac/g/hps2/laurenat/test/test_new_simp_KalVtx_wtruth.root -t 0 -y 2016

with these uncommented

Reply:
hmm, maybe we need to add another driver to the recon steering file for hps-java then. sounds like we leave this like this for now.

Reply:
Yeah, it would be good to add if possible, but I don't know how to do that!